app/CMakeLists.txt

```diff
@@ -63,27 +63,37 @@ if ((NOT CONFIG_ZMK_SPLIT) OR CONFIG_ZMK_SPLIT_BLE_ROLE_CENTRAL)
   target_sources(app PRIVATE src/combo.c)
   target_sources(app PRIVATE src/behavior_queue.c)
   target_sources(app PRIVATE src/conditional_layer.c)
+  target_sources(app PRIVATE src/endpoints.c)
+  target_sources(app PRIVATE src/hid_listener.c)
   target_sources(app PRIVATE src/keymap.c)
+
+  target_sources_ifdef(CONFIG_ZMK_BLE app PRIVATE src/behaviors/behavior_bt.c)
```

Comment:
Just out of curiosity: could this line moved inside the `if (CONFIG_ZMK_BLE)` block in line 72? I am not familiar with CMake, but that block seems to have very similar purpose as this `target_sources_ifdef`.

Reply:
Indeed it should be, good catch.

Took the opportunity to move a few more "central only" files into this conditional block, that should have been moved previously. This should lighten the peripheral side image size a bit, as an added bonus.